### PR TITLE
Update documentation and add contribution process

### PR DESCRIPTION
**Pull request by Lupphes**
Update all markdown files to a new syntax
Remove the old unused tests from Makefile
Regenerate all markdown files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,19 +53,19 @@ CentOS versions currently supported are:
...
-  To download it run:
+  To download it, run the following:
 
   ```
   podman pull registry.redhat.io/rhel9/postgresql-13
   ```
 
-  To build a RHEL9 based image, you need to run Docker build on a properly
-  subscribed RHEL machine.
+  To build an RHEL9-based image, you need to run Docker build on a properly
```
Comment (zmiklank) [inline README.md:67]:
```suggestion
  To build a RHEL9-based image, you need to run Docker build on a properly
```
I would say, the 'a' was correct here, wasn't it?

File: Makefile [hunk 2]
```diff
@@ -14,7 +14,3 @@ include common/common.mk
 clean-hook: clean-versions
 
 script_env += NOT_RELEASED_VERSIONS="$(NOT_RELEASED_VERSIONS)"
-
-# Additional upgrade tests.  Not hooked into CI ATM.
-upgrade-tests: $(VERSIONS)
```
Comment (zmiklank) [inline Makefile:19]: Could you please use a separate commit for changes of this type? And also could you please add an explanation, why this part is removed, for the future reference?

**Comment by fila43** [comment]
[test]

File: 14/root/usr/share/container-scripts/postgresql/common.sh [hunk 3]
```diff
@@ -18,6 +18,8 @@ else
     export POSTGRESQL_EFFECTIVE_CACHE_SIZE=${POSTGRESQL_EFFECTIVE_CACHE_SIZE:-$effective_cache}
 fi
 
+export POSTGRESQL_LOG_DESTINATION=${POSTGRESQL_LOG_DESTINATION:-}
```
Comment (zmiklank) [inline 14/root/usr/share/container-scripts/postgresql/common.sh:21]: Could you please add distgen-generated content in a separate commit?


Also, why is common.sh adjusted only in version 14? Do you think that some of previous, already merged, commits were not dg-generated properly?

**Review by zmiklank — CHANGES_REQUESTED** [review]
Thank you for the PR. Please see my comments.

File: README.md (hunk 1 above)
Comment (Lupphes) [reply README.md:67]: Copy-paste mistake. Thank you!

File: Makefile (hunk 2 above)
Comment (Lupphes) [reply Makefile:19]: Added as new commit

File: 14/root/usr/share/container-scripts/postgresql/common.sh (hunk 3 above)
Comment (fila43) [reply 14/root/usr/share/container-scripts/postgresql/common.sh:21]: It seems that the content for 14 was not generated correctly in one of the previous commits. So @Lupphes please rebase your PR. It should fix it.

**Comment by Lupphes** [comment]
[test]

**Comment by Lupphes** [comment]
[test-all]

**Comment by Lupphes** [comment]
[test]

**Comment by zmiklank** [comment]
The help file is generated from readme, and the readme has been changed. That would require either un-do the change or update the doc test.

**Comment by zmiklank** [comment]
[test]

**Comment by Lupphes** [comment]
[test]

**Pull request merged**